Taskfile.yaml

```diff
@@ -1,6 +1,22 @@
 version: "3"
 
 includes:
-  setup:
-    taskfile: ./03-installation-and-setup/Taskfile.yaml
+  01_setup:
```

Comment:
Any reason this includes only a portion of the module 4 task files?

Seems like all or none would make more sense to me.

Reply:
Added all task resource configurations from 04 module

Reply:
Oh, just realized this is the top level taskfile. 

The reason I put the import of module 3 here is that is part of the root level readme.

For the other modules it is expected that people will navigate to the corresponding subdirectories.


I would keep this as it was before (with only the module 3 reference)

Reply:
Sure, changes removed